src/app/layout/containers/header/header.component.html

```diff
@@ -1,4 +1,6 @@
-<user-profile *ngIf="user$ | async as user" [user]="user" />
-<button (click)="logout()">
+@if (user$ | async; as user) {
+<user-profile [user]="user" />
+}
```

Comment:
Fix your formatter. This code should look as follows:
```ts
@if (user$ | async; as user) {
  <user-profile [user]="user" />
}
```

It is not a huge issue right now, but will be with bigger blocks.